barter-data/src/lib.rs

```diff
@@ -161,9 +163,9 @@ pub mod books;
 /// [`futures_usd`](exchange::binance::futures::l2::BinanceFuturesUsdOrderBooksL2Transformer).
 pub mod transformer;
 
-/// Convenient type alias for an [`ExchangeStream`] utilising a tungstenite
+/// Convenient type alias for an [`ExchangeStream`] utilizing a tungstenite
```

Comment:
Nit-pick (for future): please don't change english spelling to american (I'm English :D)